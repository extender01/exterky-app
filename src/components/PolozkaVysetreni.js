import React from "react";
import { connect } from "react-redux";
import { odstranitVysetreni } from "../actions/vysetreniActions";
import { Link } from "react-router-dom";

const PolozkaVysetreni = (props) => (
    <div>
        <p>===========================</p>
        <Link to={"/edit/" + props.id} ><h3>{props.nazev}</h3> </Link>

        <p>{props.kam} / {props.odber}</p>

        

    </div>
);

export default PolozkaVysetreni;