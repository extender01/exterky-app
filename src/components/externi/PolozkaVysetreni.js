import React from "react";
import { connect } from "react-redux";
import { odstranitVysetreni } from "../../actions/vysetreniActions";
import { Link } from "react-router-dom";

const PolozkaVysetreni = (props) => (
    <div className="seznam-vysetreni__line">
        <div className="seznam-vysetreni__square g2">
            <Link  className="polozka"  to={"/detail/" + props.id}>   
                <p className={(props.textovyFiltr && props.nazev.toLowerCase().includes(props.textovyFiltr.toLowerCase())) ? "cssTest" : undefined}>{props.nazev}</p> {/*jestli je neco napsane v hledani a jestli to odpovida nejake polozce tak se prida css trida*/}
            </Link> 
        </div>
       
        <div className="seznam-vysetreni__square g3">
        <p className={(props.textovyFiltr && props.synonyma.toLowerCase().includes(props.textovyFiltr.toLowerCase())) ? "cssTest" : undefined}>{props.synonyma}</p>
        </div>
        
        <div className="seznam-vysetreni__square g1">
            <Link className="grey" to={"/edit/" + props.id}><p>{props.kam}</p></Link>
        </div>

       
    </div>
        
);

export default PolozkaVysetreni;

