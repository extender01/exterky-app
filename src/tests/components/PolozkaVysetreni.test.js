import React from "react";
import { shallow } from "enzyme";
import PolozkaVysetreni from "../../components/PolozkaVysetreni";
import modelovaData from "../fixtures/vysetreniFixtures";

test("mel by vyrenderovat jednu z polozek modelovych dat", () => {
    const wrapper = shallow(<PolozkaVysetreni {...modelovaData[0]} />);
    expect(wrapper).toMatchSnapshot();
});