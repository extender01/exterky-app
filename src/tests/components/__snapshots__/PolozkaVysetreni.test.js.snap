// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mel by vyrenderovat jednu z polozek modelovych dat 1`] = `
<div>
  <p>
    ===========================
  </p>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      hemopexin
    </h3>
     
  </Link>
  <p>
    krnov
     / 
    serum
  </p>
</div>
`;
